chunk.cpp

```diff
@@ -6,6 +6,19 @@
 #include "./flatteningconverter.h"
 #include "./blockidentifier.h"
 
+template<typename _ValueT>
+inline void* safeMemCpy(void* __dest, const std::vector<_ValueT>& __src, size_t __len)
```

Comment:
Why the underscores? And even double underscores?

Reply:
Good point. There is no special reason. I used them because I copy-pasted the signature from original memcpy header definition. I fixed it now.